src/main/java/utils/RequestBuilder.java

```diff
@@ -0,0 +1,88 @@
+package utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import model.enumeration.HttpMethod;
+import lombok.experimental.UtilityClass;
+import model.request.*;
+import model.request.HttpRequest.HttpRequestBuilder;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static constant.RequestHeaderConstant.CONTENT_LENGTH;
+import static java.lang.Integer.parseInt;
+import static utils.QueryStringParser.*;
+
+@UtilityClass
+public class RequestBuilder {
+    public HttpRequest getHttpRequest(BufferedReader bufferedReader) throws IOException {
+        String line = "";
+        Map<String, String> requestHeaderMap = new HashMap<>();
+        HttpRequestBuilder requestBuilder = HttpRequest.builder();
+
+        while ((line = bufferedReader.readLine()) != null && !line.equals("")) {
+            String[] tokens = line.split(" ");
+            if (isFirstLine(tokens[0])) {
+                setFirstLineProperties(requestBuilder, tokens);
+                continue;
+            }
+
+            requestHeaderMap.put(tokens[0].substring(0, tokens[0].length() - 1), tokens[1]);
```

Comment:
Nit: 
```suggestion
            requestHeaderMap.put(getKey(tokens), getValue(tokens));
```
으로 하면 tokens 처리를 조금 더 이해하기 쉬울거 같아요

Reply:
오.. 왜 이런걸 놓쳤을까요 감사합니다!